ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main__4.14-nightly.yaml

```diff
@@ -38,7 +38,9 @@ tests:
     pre:
     - chain: ipi-aws-pre
     - chain: create-infra-move-ingress-monitoring-registry
-    workflow: redhat-chaos-krkn-hub-tests
+    test:
```

Comment:
depending on which gets in first we will have to edit the HCP/rosa files too 

https://github.com/openshift/release/pull/42613